spec/features/consumer/shopping/cart_spec.rb

```diff
@@ -56,6 +56,7 @@
       let(:variant) { product_tax.variants.first }
 
       before do
+        Spree::Config.set allow_backorders: false
```

Comment:
I think it's good practice if specs leave the scene as they found it. I would create an `around` block, storing the original state, setting the wanted state and resetting it after the test. If we want this in all specs, we can do it in the spec helper. Then it's explicit that you can rely on it.